### PR TITLE
made fixes to multiple areas in front and back end code.

### PR DESCRIPTION
**Pull request by zinvergo93**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+__pycache__
+.env
+node_modules
```
Comment (gleekzorp) [inline .gitignore]: Please add the `.vscode` folder to this as well.  You will also need to stop tracking that file.

Learn more about removing tracking here.  https://stackoverflow.com/questions/24290358/remove-a-folder-from-git-tracking

**Review by gleekzorp — CHANGES_REQUESTED** [review]
Here are a few things to fix up before I can look at this further.

- Please keep all the backend Python code within the `server` folder.
- Please keep all the frontend React code within the `client` folder

**Review by gleekzorp — CHANGES_REQUESTED** [review]
Great job getting those files moved back into the `client` and `server`

- We still have a `package-lock.json`, `package.json`, `pipfile`, and `pipfile.lock` file in our root.
- The only files that should be in our root is the `readme.md`, `.gitignore`, and the two folders `client` and `server`.

Review the following link to learn how to untrack files.  https://stackoverflow.com/questions/24290358/remove-a-folder-from-git-tracking

> You might not have to untrack them, you might need to move them.

**Comment by zinvergo93** [comment]
Folder issues resolved

**Review by gleekzorp — CHANGES_REQUESTED** [review]
Folders are almost ready.

`package-lock.json` and `package.json` should be in the `client` folder.  Those files are for versioning a react or nodejs app.

Once you get those into the correct folder I would really focus your time right now on the python code.  Get the environment setup, the database setup, and the routes working correct with postman.

**Comment by zinvergo93** [comment]
Made changes to get all paths running with Postman

**Review by gleekzorp — CHANGES_REQUESTED** [review]
Awesome!  Keep up the great work.

- Please move the `package.json` into the `client` folder.  The `package.json` is for React and nodeJs version controlling.  Just like the `pipfile` is for Python version controlling.
- Please create a `readme.md` inside the `server` folder so I know how to setup my environment, get my database setup, get my server running, what routes I have available, what data needs to be sent for those routes, etc.
- I would recommend adding the `app.sqlite` file to the `.gitignore` file

File: server/readme.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+To start server:
+
+1. In your terminal, type "cd server" to enter the correct directory to get this to work.
+
+2. Enter pypi environment by typing "pipenv shell"
+
+3. Once successful, you can enter the server by running the file. In your terminal, type "python app.py"
```
Comment (gleekzorp) [inline server/readme.md]: How do I setup my database?

**Review by gleekzorp — CHANGES_REQUESTED** [review]
Looking good.  Here are some extra changes needed before I can test out the `server`.

- I still have a `package.json` inside the `server` folder.  This file needs to live inside the `client` folder.
- The `client` folder has a `Pipfile.lock` file which shouldn't be there.
- I still have an `app.sqlite` file in my `server` folder.  Please look at previous reviews for the link on how to stop tracking.
- `server` `readme` needs information on how to create the `database`.  For instance `from app import db`... etc

Below I have a screenshot of what my folder structure looks like when I pull your current changes.

move-delete-files

File: server/readme.md [hunk 3]
```diff
@@ -4,4 +4,12 @@ To start server:
 
 2. Enter pypi environment by typing "pipenv shell"
```
Comment (gleekzorp) [inline server/readme.md:6]: We might want to add a step to install the packages before hopping into the `python` repl.

I got the following error

> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/gleek/Desktop/development/tests/students/zac-git-fix-me/server/app.py", line 1, in <module>
    from flask import Flask, request, jsonify
ModuleNotFoundError: No module named 'flask'

**Review by gleekzorp — CHANGES_REQUESTED** [review]
Please update the readme with the comment listed below.

File: server/readme.md (hunk 3 above)
Comment (gleekzorp) [reply server/readme.md:6]: We only need to use `pipenv install` and that will install all the packages that are listed in the `pipfile`

**Review by gleekzorp — CHANGES_REQUESTED** [review]

**Review by gleekzorp — CHANGES_REQUESTED** [review]
Everything was setup correct.  One minor change in the `server/readme.md` for letting the user know an `app.sqlite` file will be created during step #5.

- Update the `server/readme.md` with a `Routes` section explaining what routes are available.  Such as `GET`, `POST`, etc.  Also maybe let the user know what data needs to be sent during these routes.

> Example:

```
Method: POST
URL: http://localhost:5000/add-book
Content-Type: json
Body: {
    "title": "test 1",
    "author": "test 1",
    "url": "test 1",
    "genre": "test 1",
    "star_rating": "1",
    "book_read": false
}
```

**Review by gleekzorp — CHANGES_REQUESTED** [review]
Awesome!  I believe the database is all setup now.

Let's start working on getting the client up and going now.

- Please add a `readme.md` in the `client` folder with a description on how I can get the client up and running.
- Your more than welcome to commit any changes required in the client at the same time.

**Review by gleekzorp — CHANGES_REQUESTED** [review]
Everything seems to be working.  Please update the following.

- Please add a `readme.md` inside your `client` folder with information on how to get the client up and running.
- Getting the following warning:
```
webpackHotDevClient.js:138 src/book.js
  Line 43:23:  Using target="_blank" without rel="noreferrer" is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener  react/jsx-no-target-blank
```
- Getting a `console.log('hi')` from `book.js:22`

**Review by gleekzorp — APPROVED** [review]
Everything is working great.  I'm a little worried about the `pack-lock.json` but we will work on that if it breaks.

**Pull request merged**